.github/workflows/main.yml

```diff
@@ -17,26 +17,35 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
+    # Since github actions is a mess, I have not other way of having proper compilers
+    # but to install them every time the job is run
```

Comment:
This comment looks obsolete, either fix it according to the situation, or remove completely. Also add a info block on the version difference between your local compilers and CI compilers.